### PR TITLE
Fix TxViewer when no `to` (contract deployment)

### LINKED ISSUE #4799: TxViewer re-submitting contract fails
```
eth_sendTransaction([{"from":"<removed>","to":"0x","nonce":"0x0","value":"0x0","data":"0x606060405234610000575b60008054600160a060020a03191633600160a060020a03161790555b5b610190806100366000396000f300606060405263ffffffff60e060020a6000350416630900f0108114610045578063445df0ac146100605780638da5cb5b1461007f578063fdacd576146100a8575b610000565b346100005761005e600160a060020a03600435166100ba565b005b346100005761006d61012d565b60408051918252519081900360200190f35b346100005761008c610133565b60408051600160a060020a039092168252519081900360200190f35b346100005761005e600435610142565b005b6000805433600160a060020a03908116911614156101275781905080600160a060020a031663fdacd5766001546040518263ffffffff1660e060020a02815260040180828152602001915050600060405180830381600087803b156100005760325a03f115610000575050505b5b5b5050565b60015481565b600054600160a060020a031681565b60005433600160a060020a039081169116141561015f5760018190555b5b5b505600a165627a7a7230582046c837d8b8391beea3ca635cba119de618813eadaec7ddff2214888d1711b6db0029","gasPrice":"0x174876e801","gas":"0x2dc6c0"}]): -32602: Invalid params index.e19a53db46.js:1:344466
```
When `to` is empty, re-submitting transaction leads to `to` field set to `0x` and it's rejected with invalid params message.

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes #4799

### CODE REVIEW

File: js/src/dapps/localtx/Transaction/transaction.js [hunk 1]
```diff
@@ -256,23 +259,29 @@ export class LocalTransaction extends BaseTransaction {
 
     const newTransaction = {
       from: transaction.from,
-      to: transaction.to,
```
Comment (tomusdrw) [inline js/src/dapps/localtx/Transaction/transaction.js]: That doesn't seem to be right. What if the transaction actually has valid `to`?

Comment (tomusdrw) [inline js/src/dapps/localtx/Transaction/transaction.js]: Ah, just noticed that we're adding it later on. I don't understand how does it fix the problem though. Previously `transaction.to` was set to `0x` so we will set it anyway, no?

Comment (ngotchac) [inline js/src/dapps/localtx/Transaction/transaction.js]: It's added after :) (L283)

Comment (ngotchac) [inline js/src/dapps/localtx/Transaction/transaction.js]: So the issue was that the `to` field was set, and thus formatted to `hex`: `'' => '0x'`.
I guess the API should be modified in order to take this empty `to` case into account.

Comment (jacogr) [inline js/src/dapps/localtx/Transaction/transaction.js]: Yeap. Current fix does the trick, i.e. not encoding it.

**Pull request merged**